introduction.md

```diff
@@ -4,3 +4,12 @@ Introduction
 Another apprentice to the dojo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?